rfcs/0000-central-station.md

```diff
@@ -0,0 +1,507 @@
+- Feature Name: central_station
+- Start Date: 2022-03-07
+- RFC PR: [mbta/technology-docs#0000](https://github.com/mbta/technology-docs/pull/0000)
+- Asana task: [asana link](https://app.asana.com/)
+- Status: Proposed
+
+# Summary
+
+[summary]: #summary
+
+"CentralStation" (aka "Superloc", but official name TBD) is an application that manages all of the
+MBTA's realtime data across modes. This application will combine the responsibilities of parts of
+Busloc, parts of RTR, all of CRB and Trainloc, evenutally obviate Concentrate and Realtime
+Dashboard, and be the API that Keolis, Glides, and Skate uses for Service Changes. It will be the
+canonical source of all realtime data that CTD deals with, public and private, will publish the
+standard, public data as GTFS-rt and provide API's to internal apps for manipulating data or
+retrieving non-public and non-standard data.
+
+Current realtime data ecosystem:
+
+![Current setup](imgs/realtime_current.png)
+
+Proposed realtime data ecosystem:
+
+![Proposed setup](imgs/realtime_proposal.png)
+
+# Motivation
+
+[motivation]: #motivation
+
+As was brought to light in the TRC team structure iteration plan, realtime data has until now been
+scattered across various apps, and passed around via standard and non-standard GTFS-esque files.
+With a renewed focus on the realtime data, and one team to manage it, it makes sense to
+consolidate the data into one app as well.
```

Comment:
> having all the data go through a single database-backed application didn't perform well

Since we have been talking about databases a lot on TRC, and definitely want to avoid repeating the mistakes of history, I'm curious how this system was implemented and what the performance obstacles were. Is there old documentation on it anywhere?